tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -232,9 +248,13 @@ std::unique_ptr<IStorageFactory> TestSetup::GetStorageFactory() {
 
 #ifdef USE_S3_OBJECT_STORE
   if (GetReplicaConfig().isReadOnly) {
+    if (rorPerfMode_) {
+      return std::make_unique<concord::test::ror_perf::StorageFactory>(dbPath.str(), s3OpDelay_);
+    }
     if (s3ConfigFile_.empty()) throw std::runtime_error("--s3-config-file must be provided");
     const auto s3Config = ParseS3Config(s3ConfigFile_);
-    return std::make_unique<v1DirectKeyValue::S3StorageFactory>(dbPath.str(), s3Config);
+    // return std::make_unique<v1DirectKeyValue::S3StorageFactory>(dbPath.str(), s3Config);
```

Comment:
shouldn't we leave S3StorageFactory here?

Reply:
True, this has slipped through. I just fixed it.